Index: libmatheval/tests/functions.at
===================================================================
--- libmatheval.orig/tests/functions.at
+++ libmatheval/tests/functions.at
@@ -329,13 +329,13 @@ AT_DATA([function.scm],
 
 AT_CHECK([matheval.sh function.scm], [ignore], [(1/(cosh(x)^2))], [ignore])
 
-AT_DATA([function.scm],
-[[
-(define f (evaluator-create "coth(x)"))
-(display (evaluator-evaluate-x f 1))
-]])
-
-AT_CHECK([matheval.sh function.scm], [ignore], [1.3130352854993315], [ignore])
+# AT_DATA([function.scm],
+# [[
+# (define f (evaluator-create "coth(x)"))
+# (display (evaluator-evaluate-x f 1))
+# ]])
+# 
+# AT_CHECK([matheval.sh function.scm], [ignore], [1.3130352854993315], [ignore])
 
 AT_DATA([function.scm],
 [[
